app/src/main/java/org/mifos/selfserviceapp/ui/adapters/ShareAccountsListAdapter.java

```diff
@@ -98,6 +109,15 @@ private LayerDrawable setCircularBackground(int colorId) {
         @BindView(R.id.iv_status_indicator)
         CircularImageView iv_status_indicator;
 
+        @BindView(R.id.ll_account_detail)
+        LinearLayout ll_accountDetail;
```

Comment:
use a proper naming convention like ``llAccountDetail`` and change below variables too